docs/ml-features.md

```diff
@@ -654,7 +654,11 @@ for expanded in polyDF.select("polyFeatures").take(3):
 `StringIndexer` encodes a string column of labels to a column of label indices.
 The indices are in `[0, numLabels)`, ordered by label frequencies.
 So the most frequent label gets index `0`.
-If the input column is numeric, we cast it to string and index the string values.
+If the input column is numeric, we cast it to string and index the string 
+values. When following pipeline components such as `Estimator` or 
```

Comment:
Thank you for comment and sorry for bothering you. 
Though this might be a nit-picky, it is a little difficult to infer the fact that outputs of each estimator are passed to inputs of next estimator. This doesn't seem to be described in `ml-guide.md`. ([Details](https://spark.apache.org/docs/latest/ml-guide.html#details) might be written about it.) So the reason why I wrote this sentence was to make it explicit. 
If there is any other good place to put this sentence in, we can write.

Anyway I'll update based on your pointing out.